apis/projectcontour/v1/httpproxy.go

```diff
@@ -481,7 +491,7 @@ type Route struct {
 	// Conditions are a set of rules that are applied to a Route.
 	// When applied, they are merged using AND, with one exception:
 	// There can be only one Prefix MatchCondition per Conditions slice.
-	// More than one Prefix, or contradictory Conditions, will make the
+	// More than one Prefix or Regex, or contradictory Conditions, will make the
```

Comment:
Not directly related to your PR, but noticed we're missing exact here:
```suggestion
	// More than one Prefix, Exact or Regex, or contradictory Conditions, will make the
```